app/src/main/java/org/oppia/app/databinding/EditTextBindingAdapters.java

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.databinding;
+
+import android.text.TextWatcher;
+import android.widget.EditText;
+
+import androidx.annotation.NonNull;
+import androidx.databinding.BindingAdapter;
+
+/**
+ * Holds all the custom binding adapters that bind to [EditText] adapters.
```

Comment:
How about just: ``... that bind to [EditText].``

Ditto elsewhere.

Reply:
Done!